packages/mobile/src/transactions/feed/TransactionFeed.test.tsx

```diff
@@ -46,20 +46,6 @@ const MOCK_RESPONSE: { data: { tokenTransactionsV2: { transactions: TokenTransac
           transactionHash: '0x544367eaf2b01622dd1c7b75a6b19bf278d72127aecfb2e5106424c40c268e8b',
           type: TokenTransactionTypeV2.Received,
         },
-        {
-          __typename: 'TokenTransferV2',
-          address: '0xd68360cce1f1ff696d898f58f03e0f1252f2ea33',
-          amount: {
-            tokenAddress: '0xrandomAddress',
-            value: '0.1',
-          },
-          block: '8648978',
-          fees: [],
-          metadata: {},
-          timestamp: 1542306118,
-          transactionHash: '0x544367eaf2b01622dd1c7b75a6b19bf278d72127aecfb2e5106424c40c268e8b',
-          type: TokenTransactionTypeV2.Received,
-        },
```

Comment:
Why removing this one?

Reply:
This one was supposed to be an item with an address that the app doesn't recognize, so we weren't showing it in the feed. In this PR I'm changing it to show whatever comes from blockcchain-api, so any filtering we want should be made there.